From: SeongJae Park <sj@kernel.org>
Date: Wed, 15 Oct 2025 17:27:27 -0700
Subject: [PATCH] ==== misc cleanup ====

mm/damon: misc cleanups

Yet another batch of misc cleanups and refactoring for DAMON code,
tests, and documents.

First two patches (1and 2) rename DAMOS core filters related code for
readability.

Three following patches (3-5) refactor page table walk callback
functions in DAMON, as suggested by Hugh and David, and I promised.

Next two patches (6 and 7) refactor DAMON core layer kunit test and
sysfs interface selftest to be simple and deduplicated.

Final two patches (8 and 9) fix up sphinx and grammatical errors on
documents.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_npicx3q7 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_npicx3q7

diff --git a/hkml_cv_bogus/hkml_cv_bogus_npicx3q7 b/hkml_cv_bogus/hkml_cv_bogus_npicx3q7
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.47.3

